pcr7data/policy-2/pubkeys/luks-snakeoil.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
do we have a policy-1 that we're not using anymore? or should this be policy-1 ?